Automattic/wp-calypso
# Add/config flags storage update

Related to Automattic/martech#1509

## Proposed Changes

* Adding a feature flag for storage add-on. E.g. for conditional visibility, testing, and to guard this feature until ready.
* Adding a selector to check the flag. For now it's just checking the config flag, but we will extend this with other conditions, e.g. certain plans if needed, and it makes it easier in case the feature flag gets renamed, or the logic changes otherwise. I didn't add any tests for it, because it's just taking a boolean from the config object and returns it, and casting it to boolean via `!!` (since it's reading from an untyped JSON file). 

## Testing Instructions



* Check that all the environments have this flag, and that it's set to `false`.
* For manual testing, you can also import the selector anywhere you'd like, e.g. the plans page, and `console.log` it, or add a `debugger;` statement inside the selector and verify that the value is read correctly (in this case it should return `false`).

## Pre-merge Checklist





Resolves Automattic/martech#1509
